app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  has_many :comments, dependent: :destroy
+  validates :title, :author, :text, {presence: true}
```

Comment:
You shouldn't need to use curly brackets here. Rails will assume you are writing out a hash if the last argument(s) of a method are key-value pairs.